src/components/Tool/FixEmailAddresses/FixEmailAddresses.tsx

```diff
@@ -59,7 +59,6 @@ const SourceSelect = styled(Select)(() => ({
 
 const ConfirmButton = styled(Button)(({ theme }) => ({
   backgroundColor: theme.palette.mpdxBlue.main,
```

Comment:
You can remove the backgroundColor, color and hover and just use variant='contained' on the button.